addons/frag/functions/fnc_addClassBlacklist.sqf

```diff
@@ -0,0 +1,49 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * Blacklist an ammo class, thus preventing it from producing fragments or spall. This function will not allow you to force a projectile to produce fragments or spall.
+ * The function takes three arguments, although the last two are optional. The first argument is the ammo class name to ignore, while the second and third are
+ * whether the class should be blacklisted from producing fragments and spall, respectively. Once an ammo has been blacklisted, it can be removed from the blacklist
```

Comment:
This should be placed in the "arguments" section.
The two sentences here are useless imo, last sentence is useful info ('Once an...').

Reply:
I always felt confused and under explained when reading the ACE documentation, but that's probably a me thing. I'll move relevant information to the arguments/example and delete the rest.